src/App.js

```diff
@@ -8,22 +9,35 @@ class App extends Component {
     super()
     this.state= {
       movies: [],
+      loginData: {},
       error: ''
     }
   }
 
+  updateLoginData = (loginData) => {
+    this.state.loginData = loginData;
+  }
+
   componentDidMount() {
     fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
```

Comment:
Going to be moving this into a separate file, that handles API calls. It should look neater that way.